sources/api/netdog/src/net_config/v1.rs

```diff
@@ -53,6 +53,14 @@ impl Interfaces for NetConfigV1 {
         !self.interfaces.is_empty()
     }
 
+    fn interfaces(&self) -> Vec<InterfaceId> {
+        let mut interfaces = Vec::with_capacity(self.interfaces.len());
+        for (name, _) in &self.interfaces {
+            interfaces.push(name.clone().into())
+        }
+        interfaces
```

Comment:
nit: would this be equivalent?

```suggestion
        self.interfaces.to_vec()
```

Reply:
`interfaces` is a map, so I still need to iterate through them.  That being said, I'll use an iterator chain and make this a bit simpler and shorter